Doc/library/cmath.rst

```diff
@@ -229,8 +229,9 @@ Classification functions
    tolerance is ``1e-09``, which assures that the two values are the same
    within about 9 decimal digits.  *rel_tol* must be greater than zero.
 
-   *abs_tol* is the minimum absolute tolerance -- useful for comparisons near
-   zero. *abs_tol* must be at least zero.
+   *abs_tol* is the absolute tolerance; it must be nonnegative.  Only ``0.0``
+   is close to ``0.0`` per default.  Pass an appropriate absolute tolerance to
+   compare with ``0.0``.
```

Comment:
The initial PR is my suggestion in https://github.com/python/cpython/issues/126165#issuecomment-2448116678 that replaced 2 sentences with 3:
"*abs_tol* is the minimum absolute tolerance; it must be at least ``0.0``.
When comparing to ``0.0``, the default *abs_tol* of ``0.0`` means that only ``0.0`` is close to ``0.0``.
Pass a larger tolerance to allow for floating point inaccuracies."
I respond to several comments elsewhere by discussing changes to each sentence separately and finish with a suggested rewrite of all 3.

Sentence 1: removing 'minimum', carried over from the original, looks fine to me.  'nonnegative' is equivalent to 'at least 0.0'.  In English, the latter means 'equal to or greater than 0.0' ('>= 0.0').  Preference here is partly a matter of taste and judgment of what is better for readers with a non-English first language.

Sentence 2: Since `isclose(x, 0)` is computed as `abs(x) <= rel_tol * abs(x)`; the result is False for x != 0 as long as rel_tol < 1.  I implicitly assumed this since if rel_tol >= 1, The result is True for any x.  The initial replacement of "abs_tol of 0.0" by "tolerance values" acknowledges dependence on a sane rel_tol at the cost of removing the default and being more confusing.  Specifying the default is part of the purpose of the sentence as I wrote it.  I think my original phrase was better.  I also dislike the replacement sentence "Only ``0.0`` is close to ``0.0`` per default." To me, "per default" is an awkward legalism.

Sentence 3: My suggest is a bit awkward.  In the suggested replacement, "Pass an appropriate absolute tolerance to compare with ``0.0``.", 'positive' is missing and 'to compare with 0.0' is redundant as this is the context started in the sentence before.

Here is a new abs_tol entry that takes into account the comments above.  Among other things, it separates statement of the default from discussion of closeness to 0.   I will work best with changes elsewhere that I will propose in my review comment. The markup may need tweaks.
```suggestion
   *abs_tol* is the absolute tolerance; it defaults to ``0.0`` and it must be nonnegative.  When comparing ``x`` to ``0.0``, ``isclose(x, 0)`` is computed as ``abs(x) <= rel_tol  * abs(x)``, which is false for any ``x`` and rel_tol less than ``1.0``.  So add an appropriate positive abs_tol argument to the call. 
```

Reply:
With respect to point 2, I think rel_tol >= 1 is likely a mistake of entering a percentage, such as ``1`` when ``.01`` was meant.  So we might say that it must be nonnegative and normally is much less than 1.

Reply:
I've made a few slight formatting changes to your suggestion:

```diff
- less than one
+ less than ``1``
```

```diff
- which is false for
+ which is ``False`` for
```

Reply:
Looks good.  I will wait a day before merging.